serve.js

```diff
@@ -9,13 +9,25 @@
 var http = require("http");
 var resume = require("resume-schema").resumeJson;
 var theme = require("./index.js");
+var path = require("path");
+var fs = require("fs");
 
 var port = 8888;
 http.createServer(function(req, res) {
-    res.writeHead(200, {
-        "Content-Type": "text/html"
-    });
-    res.end(render());
+    var picture = resume.basics.picture;
+    if (picture && req.url.replace(/^\//, "") === picture.replace(/^.\//, "")) {
+        var format = path.extname(picture);
```

Comment:
Yes you're right to point that out. 
It won't work because I assume the picture path to be the one related to the file system of the local server machine.
So in this interpretation `/static/images/my-picture` means that the file is in the root directory. Without the restrictive condition `req.url.replace(/^\//, "") === picture.replace(/^.\//, "")`, then  `fs.readFileSync(picture)` would make the server crash with the Node.js error `Error: ENOENT: no such file or directory`.
I understand that this not optimal and will update the pull request so that the server will both accept a picture such as `/static/images/my-picture` and catch any  `Error: ENOENT: no such file or directory` to avoid any crash when the picture is not there.